githooks/pre-commit

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# An example hook script to verify what is about to be committed.
+# Called by "git commit" with no arguments.  The hook should
+# exit with non-zero status after issuing an appropriate message if
+# it wants to stop the commit.
+#
+# To enable this hook, rename this file to "pre-commit".
```

Comment:
This file is already named `pre-commit`.

Reply:
Fail! Someone told me that already at some stage recently while working on the githooks. Thanks man, I fixed it up in the clippy PR.